test/specs/happy.js

```diff
@@ -604,6 +604,18 @@ describe('Happy Paths', options, ({driver, pageObjects}) => {
       welcome.clickOnOpenModalButton()
       welcome.verifyTitle(welcomeCopy)
     })
+
+    it('should be able to close modal with ESC button', async () => {
```

Comment:
When we'll be able to initialise the test SDK with JSON configuration, we should add tests for `shouldCloseOnOverlayClick` option